jax_rocm_plugin/build/rocm/tools/build_wheels.py

```diff
@@ -159,6 +159,7 @@ def build_jaxlib_wheel(
         "python",
         "build/build.py",
         "build",
+        "--bazel_options=--config=debug_symbols",
```

Comment:
Can we stick this in the `build/build.py` instead rather than storing bazel arguments directly in here?